examples/appoptics-config.yaml

```diff
@@ -0,0 +1,26 @@
+#
+# **This is an example configuration file. Uncomment the lines that need to be enabled.**
+#
+#
+# Collector: collector.appoptics.com:443  # - env var: APPOPTICS_COLLECTOR
+ServiceKey: your_service_key:your_service_name  # - env var: APPOPTICS_SERVICE_KEY
+# TrustedPath: /collector.crt  # - env var: APPOPTICS_TRUSTEDPATH
+# CollectorUDP: no-such-url.appoptics.com  # - env var: APPOPTICS_COLLECTOR_UDP
+# ReporterType: ssl  # - env var: APPOPTICS_REPORTER
+# Sampling:
+#   TracingMode: disabled  # - env var: APPOPTICS_TRACING_MODE
+#   SampleRate: 100  # - env var: APPOPTICS_SAMPLE_RATE
```

Comment:
Let's not advertise sample rate :)

Reply:
Removed this one and some others which I don't think the users need to fiddle with either. :)